content/07.pathogenesis.md

```diff
@@ -1,8 +1,16 @@
 ## Pathogenesis
 
-### Mechanism of Host Infection by SARS-CoV-2
-
-This section would also be great for the introduction of zoonotic diseases which has been shown to be the origin of SARS-CoV2.
+Pathogenesis is made up of three major components: entry, replication, and spread [@isbn:0-9631172-1-1 {chap. 45}].
+Based on a combination of existing research in SARS-CoV and new research in SARS-CoV-2, the SARS-CoV-2 virus was hypothesized early on to enter target cells by binding, via the viral spike (S) protein, to a specific cellular receptor.
+Subsequently the S protein is primed and cleavaged by cellular proteases [@doi:10/cf8chz; @doi:10.1016/j.antiviral.2020.104792].
+Angiotensin-Converting Enzyme 2 (ACE2) and Transmembrane Serine Protease 2 (TMPRSS2) have been identified as a prime receptor and a critical protease, respectively, facilitating SARS-CoV/CoV-2 entry into a target cell [@doi:10.1038/nature02145; @doi:10.1126/science.abb2507; @doi:10.1128/JVI.01542-10; @doi:10.1016/j.cell.2020.02.052; @doi:10.1128/JVI.02232-10].
+This finding has led to a hypothesized role for ACE2 and TMPRSS2 expression in determining which cells, tissues and organs are most likely to be infected by SARS-CoV-2.
+Based on a combination of existing research in SARS-CoV and new research in SARS-CoV-2, these insights have helped to identify which tissues and organs are most prone to SARS-CoV-2 infection.
+<!-- Needs more work, but goal is to address the questions:
+1. How does it get in a cell?
+2. How doe it get away with replicating and evading the host immune response?
+3. Where/how does it spread?
+-->
 
 #### Primary Transmission and Viral Entry
```

Comment:
Note the jump from `##` to `####` header levels will probably break numbering

Reply:
I think this is addressed [in changes here](https://github.com/greenelab/covid19-review/pull/620/commits/9f20d4d135f6d7083d83976158ef15644a7cdd84)